gsa/src/web/components/icon/importicon.js

```diff
@@ -0,0 +1,38 @@
+/* Copyright (C) 2018 Greenbone Networks GmbH
+ *
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import React from 'react';
+
+import SvgIcon from './svgicon';
+
+import {ReactComponent as Icon} from './svg/import.svg';
+
+export const ImportIcon = ({active = true, ...props}) => {
+  return (
+    <SvgIcon
+      {...props}
+    >
+      <Icon/>
+    </SvgIcon>
+  );
+};
```

Comment:
Can be simplified
```js
export const ImportIcon = props => (
  <SvgIcon
    {...props}
  >
    <Icon/>
  </SvgIcon>
);

Reply:
Could still be simplified but that's ok.

Reply:
Arg! That 'return' is just invisible in my brain... I'm going to make another commit for this in a future Misc-PR.